examples/vite/src/components/Catppuccin.jsx

```diff
@@ -1,26 +1,34 @@
+import { Color } from "./Color"
+
+const colors = [
+  "rosewater",
+  "flamingo",
+  "pink",
+  "mauve",
+  "red",
+  "maroon",
+  "peach",
+  "yellow",
+  "green",
+  "teal",
+  "sky",
+  "sapphire",
+  "blue",
+  "lavender",
+]
+
 export const Catppuccin = () => {
   return (
-    <div id="card" className="from-ctp-mantle to-ctp-crust outline-ctp-pink">
-      <h1 className="from-ctp-pink to-ctp-mauve">Catppuccin</h1>
-      <p className="text-ctp-subtext0">Soothing pastel theme for TailwindCSS</p>
+    <div id="card" className="from-mantle to-crust outline-pink">
+      <h1 className="from-pink to-mauve">Catppuccin</h1>
+      <p className="text-subtext0">Soothing pastel theme for TailwindCSS</p>
       <div id="palette">
-        <div className="bg-ctp-rosewater"></div>
-        <div className="bg-ctp-flamingo"></div>
-        <div className="bg-ctp-pink"></div>
-        <div className="bg-ctp-mauve"></div>
-        <div className="bg-ctp-red"></div>
-        <div className="bg-ctp-maroon"></div>
-        <div className="bg-ctp-peach"></div>
-        <div className="bg-ctp-yellow"></div>
-        <div className="bg-ctp-green"></div>
-        <div className="bg-ctp-teal"></div>
-        <div className="bg-ctp-sky"></div>
-        <div className="bg-ctp-sapphire"></div>
-        <div className="bg-ctp-blue"></div>
-        <div className="bg-ctp-lavender"></div>
+        {colors.map(color => (
+          <Color color={color} key={color} />
+        ))}
       </div>
       <a href="https://github.com/nekowinston/catppuccin-tailwindcss">
```

Comment:
since you're already making changes here, please could you update this to point to `catppuccin/tailwindcss`? thanks!